app/src/main/kotlin/edu/stanford/bdh/engagehf/navigation/components/AccountDialog.kt

```diff
@@ -0,0 +1,191 @@
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.shape.CircleShape
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.AccountCircle
+import androidx.compose.material.icons.filled.Close
+import androidx.compose.material3.CircularProgressIndicator
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.Icon
+import androidx.compose.material3.IconButton
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Surface
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import androidx.compose.ui.window.Dialog
+import androidx.compose.ui.window.DialogProperties
+import edu.stanford.bdh.engagehf.R
+import edu.stanford.bdh.engagehf.navigation.screens.Action
+import edu.stanford.bdh.engagehf.navigation.screens.AppTopBar
+import edu.stanford.spezi.core.design.component.VerticalSpacer
+import edu.stanford.spezi.core.design.theme.Colors
+import edu.stanford.spezi.core.design.theme.Colors.onPrimary
+import edu.stanford.spezi.core.design.theme.Colors.primary
+import edu.stanford.spezi.core.design.theme.Sizes
+import edu.stanford.spezi.core.design.theme.Spacings
+import edu.stanford.spezi.core.design.theme.SpeziTheme
+import edu.stanford.spezi.core.design.theme.TextStyles.bodyMedium
+import edu.stanford.spezi.core.design.theme.TextStyles.headlineMedium
+import edu.stanford.spezi.core.design.theme.ThemePreviews
+import edu.stanford.spezi.core.design.theme.lighten
+
+@Composable
+fun AccountDialog(appTopBar: AppTopBar, onAction: (Action) -> Unit) {
+    Dialog(
+        onDismissRequest = {
+            if (!appTopBar.isHealthSummaryLoading) {
+                onAction(Action.ShowDialog(false))
+            }
+        },
+        properties = DialogProperties()
+    ) {
+        Surface(
+            shape = MaterialTheme.shapes.medium,
+            color = MaterialTheme.colorScheme.surface.lighten(),
+            modifier = Modifier
+                .fillMaxWidth()
+                .padding(Spacings.medium)
+        ) {
+            Column(
+                modifier = Modifier.padding(Spacings.medium),
+                horizontalAlignment = Alignment.CenterHorizontally,
+            ) {
+                Box(
+                    modifier = Modifier.fillMaxWidth(),
+                    contentAlignment = Alignment.TopStart
+                ) {
+                    IconButton(
```

Comment:
> Typically the convention for close buttons is to place them on the top right. Also the it should be the only element in the row and the rest of the elements should be below it

As a left-hander, I regularly notice how stupid it is to reach the X because it's on the left side, but I've just double-checked that for instance Account in Gmail, Google Photos, etc:
![image](https://github.com/user-attachments/assets/dc92830b-7878-484a-9f75-dae3dc8e1bc5)

as a left-hander, right-handed is even more comfortable, as already mentioned, but I personally don't have a strong opinion on it, if you prefer it on the right I can change it if you like, @eldcn  and @PSchmiedmayer 

I also saw that there is usually a title, so I added that and moved the content to the bottom as you suggested 


![image](https://github.com/user-attachments/assets/c2b86b73-abf0-46d1-8794-014e39718596)